lib/assessment.js

```diff
@@ -8,32 +8,73 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
+var min = function min(previousValue, currentValue, index, array) {
+    var result = previousValue;
+    if(currentValue < result) result = currentValue;
+    return result;
+
 
 };
 
+var array = [5, 3, 2, 4]
+console.log(array.reduce(min))
+
 // Question 2
 // Write a callback for reduce that produces
 // the sum of the values in the array
-var sum = function sum() {
+var sum = function sum(previousValue, currentValue, index, array) {
+       var result = previousValue;
+     result += currentValue;
+    return result;
+
 
 };
 
+var array = [5, 74, 2, 4]
+console.log(array.reduce(sum));
+
 // Question 3
 // Write a function that returns true
 // if invoking the callback for any element
 // in the array returns a truthy value
 var any = function any(array, callback, thisArg) {
 
+ var result = true;
+
+ array.forEach(function(currentValue, index, array){
+     result = result && callback(currentValue, index, array);
+ });
+ return result;
+
+};
+
+var numbers = [6, 8, 26, 80];
+
+var even = function(currentValue) {
+  return (numbers % 2) === 0;
```

Comment:
In these callbacks intended for any, you're trying to access the array `numbers` through an outside reference rather than using the passed-in value. Not only do they not work, but if they did, they'd operate on an array other than the one that was passed in to `any` for it to operate on.